addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -326,6 +326,10 @@
                 decoration-warning="state == 'cancel' or (json_popover and 'text-warning' in json_popover)"
                 color="workcenter_id"
                 display_unavailability="1"
+                progress_bar="workcenter_id"
+                total_row="True"
+                js_class="mrp_workorder_gantt"
```

Comment:
As the js class `mrp_workorder_gantt` doesn't exist in community, I think you should add this attribute in `mrp_workorder`.
Same for line 372

Reply:
The gantt workorder views are defined in community but the gantt view itself is only available in enterprise edition.
I  see no interest of defining a new view simply for an attribute, except to be consistent and move all the enterprise features (gantt, grid, cohort, dashboard and map views) into the enterprise part.